src/js/providers/flash.js

```diff
@@ -338,6 +338,10 @@ define([
                         this.trigger(events.JWPLAYER_MEDIA_ERROR, event);
                     }, this);
 
+                    _swf.on('adaptiveEventAudio', function() {
+                        this.trigger('mediaType', {mediaType: 'audio'});
+                    }, this);
```

Comment:
If you send this, then I also expect you to send `{mediaType: 'video'}`. Every provider in Flash needs to be updated to output video or audio when a new item is played. Otherwise, in mixed playlist, after an audio only stream, 'mediaType' will be stuck on 'audio' and never update.